src/main/java/org/kitteh/irc/client/library/implementation/Config.java

```diff
@@ -135,7 +135,7 @@ public String toString() {
     static final Entry<StringConsumerWrapper> LISTENER_OUTPUT = new Entry<>(null, StringConsumerWrapper.class);
     static final Entry<Integer> MESSAGE_DELAY = new Entry<>(Client.DEFAULT_MESSAGE_DELAY, Integer.class);
     static final Entry<String> NICK = new Entry<>("Kitteh", String.class);
-    static final Entry<String> REAL_NAME = new Entry<>("Kitteh", String.class);
+    static final Entry<String> REAL_NAME = new Entry<>("Kitteh v" + KICLProperties.DEFAULT_VERSION, String.class);
```

Comment:
Realnames are silently truncuated to a good 40 characters, inconsistently across IRCds, with no standard.  Keep this in mind